.backportrc.json

```diff
@@ -0,0 +1,9 @@
+{
+    "upstream": "elastic/rally-tracks",
+    "targetBranches": ["8.15"],
```

Comment:
I did some testing, and it looks like PR labels are converted to branch list based on `branchLabelMapping` which is filtered through `targetBranchChoices` which in turn gets overridden by `targetBranches`. So current configuration will ignore the labels and always backport to `8.15` regardless of the labels.

I think we need to choose between the following:
* configure `targetBranches` to the current latest branch which is closest to what we had earlier,
* configure `branchLabelMapping` and label each time (more work but more flexible).

WDYT?